docs/user_manual/dataset-terminology.md

```diff
@@ -1,12 +1,12 @@
 <!--
-SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+SPDX-FileCopyrightText: Contributors to `power-grid-model` project <powergridmodel@lfenergy.org>
```

Comment:
@Jerry-Jinfeng-Guo  Please revert this in the license header.

Reply:
Oops, auto replace

Reply:
Reverted in the latest commit.